WebView/app/src/main/assets/main.js

```diff
@@ -23,7 +23,7 @@ function sendAndroidMessage() {
 }
 
 function getData() {
-	fetch("https://gcoleman799.github.io/Asset-Loader/weather.json").then(function(resp) {
+	fetch("https://github.com/android/views-widgets-samples/tree/webview/WebView/sampleData/weather.json").then(function(resp) {
```

Comment:
Should this be `raw.githubusercontent.com` rather than `github.com` ?

Reply:
Yes, I think so. I am going to remove the change to where the data is being fetched from. This way we can test that the URL is correct before we make the change.